src/main/java/com/github/packageurl/PackageURL.java

```diff
@@ -92,8 +92,13 @@ public PackageURL(final String type, final String name) throws MalformedPackageU
      * @deprecated use {@link #PackageURL(String, String, String, String, Map, String)} instead
      */
     @Deprecated
-    public PackageURL(final String type, final String namespace, final String name, final String version,
-                      final TreeMap<String, String> qualifiers, final String subpath)
+    public PackageURL(
+            final String type,
+            final String namespace,
+            final String name,
+            final String version,
+            final TreeMap<String, String> qualifiers,
+            final String subpath)
```

Comment:
This change has something to do with the number of parameters? Maybe increase?

Reply:
The formatter limits the line to 120 characters and starts wrapping if it's more.

Reply:
I guess I was confused about why, if it doesn't wrap, it allows multiple parameters on one line, but when it wraps it only allows _one_ parameter per line.